### ISSUE TITLE
SA1012 and SA1013 should not report for interpolated strings

### ISSUE THREAD

**Issue by vweijsters** [DotNetAnalyzers/StyleCopAnalyzers#552]
The following code (from our own codebase :wink:) will  trigger SA1012 and SA1013.

```C#
context.ReportDiagnostic(Diagnostic.Create(Descriptor, xmlElementSyntax.StartTag.GetLocation(), $"The XML tag '{xmlElementSyntax.StartTag.Name}' is not closed."));
```

Obviously neither of them should have been reported.

**Comment by pdelvo** [comment]
You're right. I would grab that if @sharwell doesnt insist on doing it.

**Comment by sharwell** [comment]
@pdelvo Go for it
